### PR TITLE
PR: Add config manager stub

### PR DESCRIPTION
**Pull request by goanpeca**
## Description of Changes

* [ ] Wrote at least one-line docstrings (for any new functions)
* [ ] Added unit test(s) covering the changes (if testable)

* [ ] Included a screenshot or animation (if affecting the UI, see [Licecap](https://www.cockos.com/licecap/))

### Issue(s) Resolved

Fixes #

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: @goanpeca

### CODE REVIEW

**Comment by goanpeca** [comment]
@ccordoba12 this could be a simple idea to replace the current CONF with a config manager that would provide and change the correct configuration depending on if the user has a specific project or not open.

Thoughts @ccordoba12 ?

File: spyder/config/manager.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    def load_config(self):
+        # Main configuration instance
+        try:
+            self._global_config = UserConfig(
+                'spyder',
+                defaults=DEFAULTS,
+                load=True,
+                version=CONF_VERSION,
+                subfolder=SUBFOLDER,
```
Comment (ccordoba12) [inline spyder/config/manager.py]: Perhaps we need to change `SUBFOLDER` in case there's an active project?

Comment (goanpeca) [reply spyder/config/manager.py]: Yes correct, I am just swapping the normal CONF for this manager and make sure all works as expected. Then I will add the correcto conf.

Just wanted to double check if this made sense

**Comment by goanpeca** [comment]
>We still have to think how to update spyder.ini when we change a default option 

Hmmm I don't understand, could you expand a bit more on this?

>and what to do with workspace.ini.

I will need to a add compatibility layer to handle this

**Comment by goanpeca** [comment]
Closing this one as https://github.com/spyder-ide/spyder/pull/9820 has all the work

**Pull request closed without merging**